trafilatura/core.py

```diff
@@ -747,7 +747,7 @@ def bare_extraction(filecontent, url=None, no_fallback=False,
                 LOGGER.info('blacklisted URL: %s', url)
                 raise ValueError
             # cut short if core elements are missing
-            if only_with_metadata is True or with_metadata is True and any(
```

Comment:
`with_metadata` isn't deprecated yet. There could be a line `only_with_metadata = with_metadata` earlier in the code though.

Reply:
@felipehertzer I will take care of that.

Reply:
I had problems with cli that was not passing the variable with_metadata to the extractor() and this error should only have been called when only_with_metadata is required.  Alrgiht I'll leave it to you

Reply:
I think I fixed the CLI problems in 3eafd7e5c09c7b49c513342eb5089544f29c189e, if something still isn't working we can add tests for it.

Reply:
Line added in 968721971c0d65f9fe95b983ca050c0f97d4b926.